content/ko/docs/howto/charts_tips_and_tricks.md

```diff
@@ -41,22 +41,23 @@ value: {{ required "A valid .Values.who entry required!" .Values.who }}
 
 ## 문자열에는 따옴표를 쓰고, 정수형에는 쓰지 말자
 
-When you are working with string data, you are always safer quoting the strings
-than leaving them as bare words:
+문자열 데이터를 사용할 때에는 문자열로 그대로 두기보다 
+쌍따옴표로 값을 묶는 것이 안전합니다.:
 
 ```yaml
 name: {{ .Values.MyName | quote }}
 ```
 
-But when working with integers _do not quote the values._ That can, in many
+하지만 integer는 많은 경우에 쿠버네티스에서 파싱에러가 발생할 수 있으니 
+_쌍따옴표를 사용하지 마세요._
+when working with integers _do not quote the values._ That can, in many
```

Comment:
Hi @nuguni - it looks like this line still needs to be removed.

Reply:
Hi @angellk  I removed it. Thank you.